src/components/calcite-button/calcite-button.scss

```diff
@@ -68,6 +68,7 @@
   height: 100%;
   border-radius: 0;
   border: none;
+  gap: $baseline/3;
```

Comment:
I don't think browser support is there for this unfortunately in ie, safari, etc :(

https://caniuse.com/?search=gap

Reply:
Dah! I misread caniuse.
I see the Safari prob.
But FWIW, are we supporting IE?

Reply:
Yes, at least until after the Olympics. It also doesn't work on any mobile browser except for Chrome.

Maybe next decade..

Reply:
LOL. Updated.